scalacheck-binding/src/main/scala/scalaz/scalacheck/ScalazArbitrary.scala

```diff
@@ -537,8 +537,8 @@ object ScalazArbitrary extends ScalazArbitraryPlatform {
     Functor[Arbitrary].map(A)(LazyEitherT[F, A, B](_))
 
   // backwards compatibility
-  def stateTArb[F[+_], S, A](implicit A: Arbitrary[S => F[(S, A)]]): Arbitrary[StateT[F, S, A]] =
-    indexedStateTArb[F, S, S, A](A)
+  def stateTArb[F[+_], S, A](implicit A: Arbitrary[S => F[(S, A)]]): Arbitrary[StateT[S, F, A]] =
```

Comment:
Due to your other comment, should this also be changed to `def stateTArb[S, F[+_], A]`?

Reply:
yes!

Reply:
Other than this, looks good to me.